widgetssdk/src/main/java/com/glia/widgets/view/floatingvisitorvideoview/FloatingVisitorVideoView.java

```diff
@@ -0,0 +1,89 @@
+package com.glia.widgets.view.floatingvisitorvideoview;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.widget.FrameLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+
+import com.glia.androidsdk.comms.VideoView;
+import com.glia.widgets.R;
+
+public class FloatingVisitorVideoView extends ConstraintLayout {
+    private VideoView videoView;
+    private FrameLayout videoContainer;
+
+    public FloatingVisitorVideoView(@NonNull Context context) {
+        this(context, null);
+    }
+
+    public FloatingVisitorVideoView(@NonNull Context context, @Nullable AttributeSet attrs) {
+        this(context, attrs, 0);
+    }
+
+    public FloatingVisitorVideoView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+        this(context, attrs, defStyleAttr, 0);
+    }
+
+    public FloatingVisitorVideoView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+        init();
+    }
+
+    public void onResume() {
+        resumeVideoStream();
+    }
+
+    public void onPause() {
+        pauseVideoStream();
+    }
+
+    public void onDestroy() {
+        releaseVideoStream();
+    }
+
+    public void showVisitorVideo(VideoView newVideoView) {
+        videoView = newVideoView;
+        videoView.setZ(20);
```

Comment:
dat magic number

Reply:
Unfortunately, looks like it's the issue in the core SDK.

Reply:
removed - will add elevation stuff back later if required :)